src/shared/components/ui/index.ts

```diff
@@ -1,2 +1,25 @@
+import { Control, FieldValues, Path } from "react-hook-form";
+
 export * from "./Button";
+export * from "./Calendar";
+export * from "./DateField";
+export * from "./DatePicker";
 export * from "./Modal";
+export * from "./RangeCalendar";
+
```

Comment:
Should probably move the following types somewhere else.

Reply:
Agree, can you move this to another file please?